css/css-basics_pokemon-page/css/styles.css

```diff
@@ -59,22 +73,28 @@ section {
 */
 
 .subtitle {
-  /* your solution */
+  display: inline-block;
+  border-right: 3px solid gold;
```

Comment:
Not wrong, but there is an easier solution, so you only have one line to style the border left and right. :)